db/migrate/20190530154456_add_doi_location_to_publish_targets.rb

```diff
@@ -0,0 +1,6 @@
+class AddDoiLocationToPublishTargets < ActiveRecord::Migration[5.2]
```

Comment:
Could we rename this migration?  Doesn't seem to match what it's doing.

Reply:
From before the field rename, I'll rename it. How's "AddDoiAttributesToPublishTargets"?

Reply:
Sounds good!